test/parallel/test-domain-timers-uncaught-exception.js

```diff
@@ -0,0 +1,14 @@
+const domain = require('domain').create();
+var first = false;
+
+domain.run(function() {
+  setTimeout(function() { throw Error('FAIL'); }, 1);
+  setTimeout(function() { first = true; }, 1);
+  setTimeout(function() {
+    if (first === false) {
```

Comment:
Why can't you use `assert.strictEqual(first, true);`? Is it because you are in a domain, and the assert won't work?